tests/react-directive-spec.js

```diff
@@ -25,6 +25,22 @@ var Hello = React.createClass({
   }
 });
 
+var People = React.createClass({
+  propTypes: {
+    items : React.PropTypes.array
+  },
+
+  render(){
+    var people = this.props.items;
+    var names = people.map(function(person, index) {
+      var name = person.fname + ' ' + person.lname;
+      return (++index === people.length) ? name : name + ', ';
+    });
```

Comment:
Since items in the array are objects and i'm concatenating person.fname and person.lname I can't just use join, however I did update the code to use join for the name printing rather than using index.  Thanks!